client/js/controllers/editor.js

```diff
@@ -831,6 +831,64 @@
                     }
                 };
 
+             $scope.importFile = function () {
+                    var file = filePath;
+                    if (file) {
+                        if (isLeaf) {
+                            var cached = file.split("/");
+                            cached.pop();
+                            file = cached.join("/");
+                        }
+                        var dialog = $('<div></div>').
+                                   html($compile('<form ng-controller="editor"> <input' +
+                                                 ' type="file" class="inputControls"' +
+                                                 ' multiple onChange="angular.element(this).scope().filesChanged(this)"/> <input type="button" class="inputControls" ng-click="uploadFiles()" value="Upload" /><li ng-repeat="file in files">{{file.name}}</li></form>')($scope)).
+                        dialog({
+                          title: "Choose file to import",
+                          autoOpen: false,
+                          modal: true,
+                          position: { at: "center top"},
+                          height: 167,
+                          width: 300,
+                          show: { effect: "fade", duration: 300 },
+                          hide: {effect: "fade", duration: 300 },
+                          resizable: 'disable',
+                          buttons: {
+                              Cancel: function() {
+                                  $(this).dialog("close");
+                                  }
+                              },
```

Comment:
You can create Upload button like the Cancel one (with the same style and position).

It should be something like this: 

```
buttons: {
      "Upload": function () {
            // Do something here
       },
      Cancel: function() {
            $(this).dialog("close");
       }
},
```

Try avoiding creating the html button you've created for Upload :)